spec/index.bs

```diff
@@ -2039,17 +2042,21 @@ interface GPUBuffer {
 GPUBuffer includes GPUObjectBase;
 </script>
 
-{{GPUBuffer}} has the following internal slots:
+{{GPUBuffer}} has the following attributes:
 
 <dl dfn-type=attribute dfn-for="GPUBuffer">
-    : <dfn>\[[size]]</dfn> of type {{GPUSize64}}.
+    : <dfn>size</dfn>
     ::
         The length of the {{GPUBuffer}} allocation in bytes.
 
-    : <dfn>\[[usage]]</dfn> of type {{GPUBufferUsageFlags}}.
+    : <dfn>usage</dfn>
     ::
         The allowed usages for this {{GPUBuffer}}.
+</dl>
+
+{{GPUBuffer}} has the following internal slots:
```

Comment:
nit: I think we should have an internal slot for the buffer descriptor and have the attributes return values from it (see comment below).

Reply:
Updated the PR to do that. It certainly makes the PR itself smaller, since we don't need to update all the algorithms anymore. (I guess we still could, which may still be nice for readability.)